### ISSUE TITLE
Chapter 5 reading a dll.

### ISSUE THREAD

**Issue by riverman369** [markjprice/cs11dotnet7#4]
Please check that your issue is not already in the [errata aka list of corrections and improvements](https://github.com/markjprice/cs11dotnet7/blob/main/docs/errata/README.md). Please note that PRs will be ignored because code in the repo must match the print book already published. Please raise an issue instead so I can handle it appropriately. All issues will be entered into the errata once closed and fixed in the next book edition.

Chapter: 5
Page Number: 366-
Section Title: 
Problem to fix: output is dll not a window.
Suggested solution:
Other useful information e.g. OS, coding tools, and so on: I've tried this on a HP Pavilion laptop using the latest VS Code. and on a Desk top computer. I've gone through this particular chapter at least 5 times on each system, and still get the same result. I'm still new to programming, if I'm doing something wrong I would love to know what it is. Both systems will build the project, but when I go to run it, it gives an error. "out put is DLL now a window.

**Comment by markjprice** [comment]
In c, Chapter 5: Building Your Own Types with Object-Oriented Programming starts on page 199 and ends on page 248. You have stated page 366 which is in Chapter 8. You did not state the section title so I cannot know where you are having an issue. I also tried searching the PDF for the text "output is dll" and "not a window" and it does not appear. Please confirm which page you are having a problem with and better yet include the section title. And if you get an error then copy and paste the error text because the "Problem to fix" that you state "output is dll not a window" makes no sense.

**Comment by markjprice** [comment]
Is it possible that you are trying to run one of the class library projects, as shown in this screenshot below? You cannot run the class library projects that you create in Chapter 5. You must run the console app project named `PeopleApp`. Make sure you are in the `PeopleApp` directory when you enter `dotnet run`.

![image](https://user-images.githubusercontent.com/14040265/204372719-b3011a4f-caa7-4199-a9a8-17923fce95c3.png)

**Comment by riverman369** [comment]
Yes that is the aea I am in, and it doesn't work on either of my computers! I have made sure I was in the proper section, and all I get is an e]rror that says it can't read a dll as an output. No console comes up. Two computers cant be wrong 8 times, that is how many times ive been through this section, 4 on each computer, a laptop and a desk top running windows 10, with visual studio code on each, totally up to date as of today.

**Comment by markjprice** [comment]
You still have not told me which section you are in. I cannot help unless I know exactly what you are doing. Please tell me the section title and the step you are trying to complete. 

Two computers can be wrong 8 times if you are doing the same wrong thing 8 times. But I cannot guess what wrong thing you might be doing. Please take a screenshot.

**Comment by riverman369** [comment]
I apologize for taking so long to answer. but as I'm a truck driver, I don't get much time to work on this as I would like. I did figure out the problem, it was Microsoft's issue! On my laptop as I was redoing chapter 5, I received several updates, and everything is working fine now. My desktop, I had to delete VS Code and re-install it. now it is working fine.
and just so you know, I never said you made a mistake, you asked to be informed about an issue, so I did just that, it's not my fault that Microsoft had an issue, nor yours.

**Comment by markjprice** [comment]
No worries, I'm glad you found a fix. Best of luck with your learning.

**Comment by riverman369** [comment]
Issue fixed...I had sent Microsoft a copy of my issue, and received several
updates when I was checking for the last time how and where the problem
was. I did everything exactly as before and now it works.
So the real issue was with Microsoft's programs, not me, and not you.
Be careful how you accuse someone of making a mistake, I wasn't accusing
you. You asked to be notified about a problem, I did just that. No
accusations on my part
